sound/soc/intel/boards/sof_sdw_rt_sdca_jack_common.c

```diff
@@ -88,6 +88,10 @@ static const char * const jack_codecs[] = {
 	"rt711", "rt712", "rt713", "rt722"
 };
 
+static const char * const need_sdca_suffix[] = {
+	"rt711", "rt713"
+};
```

Comment:
why does rt713 need a suffix? There's no ambiguity or conflict with a previous generations, is there?

Reply:
> why does rt713 need a suffix? There's no ambiguity or conflict with a previous generations, is there?

Because [rt713-sdca.conf](https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/sof-soundwire/rt713-sdca.conf) is already used in UCM

Reply:
ah, ok, worthy of a comment that we're doing this to avoid breaking userspace.

"the sdca suffix is required for rt711 since there are two generations of the same chip. RT713 is an SDCA device but the sdca suffix is required for backwards-compatibility with previous UCM definitions".

Reply:
comment added. And thanks to @ujfalusi for the suggestion of matching the codec_dai->name